src/handwritten/task.py

```diff
@@ -71,6 +71,7 @@ def __init__(self, new_task_name='', *, grpc_options=None):
                 results in an error.
         """
         self._handle = lib_importer.task_handle(0)
+        self._close_on_exit = False
```

Comment:
These assignments are no longer needed. The code below will initialize these fields or raise an exception.

Reply:
Oops, `__del__` warns if `self._handle` doesn't exist. Keep the assignments, but initialize `self._handle` to `None`.

Reply:
Initialized `self._handle = None`.